src/stores/game.js

```diff
@@ -18,7 +18,7 @@ function queenCount(player) {
   if (!player?.faceCards) {
     return 0;
   }
-  return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
+  return player?.faceCards?.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0) ?? 0;
```

Comment:
Similarly to `kingCount`, this is a bit of scope creep for this PR but it should stop this function from type erroring when `player` is undefined